public/app/features/dashboard/dashgrid/PanelStateWrapper.tsx

```diff
@@ -92,7 +92,7 @@ export class PanelStateWrapper extends PureComponent<Props, State> {
     super(props);
 
     // Can this eventBus be on PanelModel?  when we have more complex event filtering, that may be a better option
-    const eventBus = props.dashboard.events.newScopedBus(`panel:${props.panel.id}`, this.eventFilter);
+    const eventBus = props.dashboard.events.newScopedBus('panel', this.eventFilter);
```

Comment:
So this change is also needed so that uPlot can retrieve a correct path of multiple scoped events buses that we create in a dashboard. Every panel has it's own ScopedEventsBus. 

I don't know why the key here was unique per panel. I think about the first argument here as a type of the bus, rather than it's key. Whatever filtering we do in the future using ScopedEventsBus, I think should be configured on eventFiter rather than based on a event's bus key.

Reply:
Judging by this comment

>>   // The path is not yet exposed, but can be used to indicate nested groups and support faster filtering

It looks like I added path as way to do nested filtering, but looks like it's not used any where. Only filter on origin (bus)